docs/acr-roadmap.md

```diff
@@ -1,117 +1,159 @@
+Customers looking to test their connectivity to their ACR can use [az acr health-check][health-check]
 
+### Auto Purge
 
-There are many additions that will come, including perf improvements and the ability to branch off production workloads while developers continue to iterate. 
+As registries are filled with automated image builds, they wind up filling with tags, manifests and layers that never get used, or were deployed for a period of time, but no longer. [Auto-purge-spec](https://github.com/AzureCR/specs/issues/1) outlines how ACR will track image usage and move unused layers to a recycle bin, allowing subsequent purging. The feature will be configured and managed, with reasonable defaults, assuring you'll never lose anything you really wanted to keep.
 
+As a step along this automated journey, [ACR Scheduled Tasks][scheduled-tasks] and a new ACR image (mcr.microsoft.com/acr/acr-cli:0.1) can be used to [automate image deletion][auto-purge].
 
-## Backlog ##
+### ACR Helm GA
 
-### Limit Endpoint Access /VNet Support ###
-Customers have asked for limitations on their registry, based on the IP, not just authentication. As a shared registry API, this does present some challenges that we'll need to address. 
+**Q: When will ACR Helm Support GA?**  
+With the collaboration of [Josh Dolitsky](https://jdolitsky.io), of [Chart Museum](https://chartmuseum.com), we added [Helm 2][helm] support to ACR - `az acr helm`. As we watched the industry look for **Y**et **A**nother **S**torage **S**olution (YASS) for new types like [Singularity][singularity], [CNAB](https://cnab.io/) and other new types, we realized we needed a more generalized solution. With help from people like [Stephen Day](https://www.linkedin.com/in/stephenjday/) and the [OCI community][oci-org], we focused on [creating a generalized Artifacts store](https://stevelasker.blog/2019/01/25/cloud-native-artifact-stores-evolve-from-container-registries/). In August of '19, the [OCI TOB](https://github.com/opencontainers/tob/pull/60) adopted [OCI Artifacts][artifacts-repo].
 
-We currently have VNet and Firewall rules available in private preview. Please contact us at acrsup@microsoft.com for being added to the VNet Private Preview. 
+With OCI Artifacts in place, we are working with the [Helm Community](https://github.com/helm/) to add [Helm Registry support](https://v3.helm.sh/docs/topics/registries/), allowing customers to use `helm registry push|pull|` across all OCI compliant Registries. As Helm 3 Registry support releases, and the community moves to Helm 3, our plan is to deprecate the Helm 2 acr support for, what we believe is a more native Helm 3 experience. We will continue to support `az acr helm` support for a period of transition to Helm 3. All Helm charts pushed to ACR with `az acr helm` will be fully compatible with Helm 3 registry support. Customers using ACR Helm should be comfortable with a smooth transition to Helm 3 and continue with `az acr helm` in the meantime.
```

Comment:
Helm 3 and 2 might be breaking and this way we would actually even deprecate az acr helm if helm client is as rich as docker.

Reply:
Not sure what exact change you're suggesting. Ping me today and I can make another PR.

Reply:
> All Helm charts pushed to ACR with `az acr helm` will be fully compatible with Helm 3 registry support. 

@SteveLasker @sajayantony Could you please clarify what this means? 

We _were_ pushing charts to ACR with Helm 3 OCI but discovered that Spinnaker [does not yet have OCI support](https://github.com/spinnaker/spinnaker/issues/5474). We are switching to pushing charts with `az acr helm` to get our Spinnaker pipelines working. We are trying to decide if we should "dual push" charts with both `az acr helm push` (helm 2) and `helm chart push` (helm 3 OCI) to be ready for when Spinnaker supports OCI. 

This comment makes me think that charts pushed with `az acr helm push` will someday be accessible with helm 3 OCI commands and also show up in the Portal in the ACR UI. Thanks!

Reply:
@ iowillhoit - Helm 2 charts aren't stored the same way as OCI charts are so we can't auto migrate the content over. You would have to migrate and repush them as Helm3 charts. We don't have a timeline for deprecation and just want to make sure the the path going forward is clear for helm2 and helm3 authors.